pkg/util/health/startup-issue-collector.go

```diff
@@ -0,0 +1,69 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
nit: go file names are typically underscored instead of dashes, could we rename this to be more uniform with the rest of the repo?